win32/reals/real_threadpool.c

```diff
@@ -3,6 +3,8 @@
 
 #include "real_gballoc_hl_renames.h"
 
+#include "real_interlocked_renames.h"

```

Comment:
>"real_interlocked_renames.h" [](http://example.com/codeflow?start=9&length=28)

<I suspect>

The amount of work which will need to follow this in order to propagate the change in UTs and everywhere will be... rather large, euphemistically talking.

</I suspect>